src/tera.rs

```diff
@@ -21,34 +21,29 @@ impl tera::Function for Localizer {
             .and_then(|key| key.as_str())
             .ok_or(tera::Error::msg("missing ftl key"))?;
 
-        let bundle = self.get_locale(&lang_arg).ok_or(tera::Error::msg(format!(
-            "locale not registered: {lang_arg}"
-        )))?;
-
-        let msg = bundle
-            .get_message(ftl_key)
-            .ok_or(tera::Error::msg(&format!(
-                "FTL key not in locale: {}",
-                ftl_key
-            )))?;
-        let pattern = msg
-            .value()
-            .ok_or(tera::Error::msg("No value in fluent message"))?;
+        let ftl_attribute = args.get("__attribute").and_then(|attr| attr.as_str());
```

Comment:
@tronicboy1 I am using `__attribute` here to avoid backwards-compatibility issues in case people already used this variable name (and I would personally feel like it would be better to have the other parameters also be `__lang` respectively `__key`), but if you want to have these unified, one could change it to `attribute` as well.

Reply:
@jreppnow 
I would say unify them as `lang`, `key`, and `attribute`. The extra underscores are annoying to type and they leave the impression that they are "private" fields. I understand your intention with trying to avoid collisions between tera variables and FTL variables.

Reply:
Done.